src/inmanta/protocol/rest/server.py

```diff
@@ -158,6 +158,7 @@ async def _call(self, kwargs: Dict[str, str], http_method: str, call_config: com
                 self.respond({"message": "Unable to decode request body"}, {}, 400)
 
             except exceptions.BaseHttpException as e:
+                LOGGER.warning("Got %d status code: %s", e.to_status(), e.to_body())
```

Comment:
This will log a warning for every non-200 response we return via our API (roughly? ) ? 

As I understand, this is on the dispatcher path, the message suggests we `Got` an exception, but it is more likely that we produce the exception. 

Also, in `_execute_call` there already is : 
```
        except exceptions.BaseHttpException:
            LOGGER.debug("An HTTP Error occurred", exc_info=True)
            raise
 ```

So we will log it twice at different level.

Reply:
I did not get errors in my logs. Everything that generates exceptions outside _execute_call will only get a warning log from the tornado with the status code and the url.

Reply:
I am open for any other solution that ensures it is possible to configure authentication based on the logs (and not having to use a debugger)

Reply:
What exactly happened?

Reply:
The aud setting was wrong, for which you get the message https://github.com/inmanta/inmanta-core/blob/master/src/inmanta/protocol/common.py#L1032

But the logs will only say: "403 /.... "

Reply:
I checked the docs https://internal.inmanta.com/development/core/technical-designs/workmanship.html?highlight=workman


Level | Meaning
-- | --
Trace | Information only relevant to developers of specific subsystems.
Debug | Detailed information, typically of interest only when diagnosing problems, intended audience is developers and support staff.
Info | Information on normal progress, should be understandable to operators and end users.
Warning | Information on potential problems, targeted towards operators and end users.
Error | Information on problems, that should be examined by an operator, but not immediately.
Critical | Information on problems that are a threat to system stability. An  operator should be notified immediately (even in the middle of the  night).

So this can be considered to be in line with that. 

I don't know if logging every 404 as a warning makes sense, but it does improve visibility

Reply:
That is the thing, tornado logs every 404 but without the message that we send to the client. That makes debugging really difficult.

Alternatively we disable the tornado logger and replace it with this one. Then there is the same amount of log line but with more information